src/Symfony/Component/Messenger/EventListener/SendFailedMessageToFailureTransportListener.php

```diff
@@ -25,12 +26,19 @@
  */
 class SendFailedMessageToFailureTransportListener implements EventSubscriberInterface
 {
-    private $failureSender;
+    /**
+     * @var ServiceLocator|SenderInterface
+     */
+    private $failureSenders;
```

Comment:
should be moved as `@param ContainerInterface $failureSenders` in the constructor docblock.